app/docs/enterprise/0.31-x/installation/centos.md

```diff
@@ -0,0 +1,17 @@
+---
+title: CentOS
+class: page-install-method
+---
+
+# Installing Kong Enterprise on CentOS
+
+A quick guide on how to install Kong Enterprise on CentOS.
+
+1. Head over to https://bintray.com/kong/kong-enterprise-edition-rpm
+2. Click **Set Me Up** and follow the instructions on the **Download > Resolving RPM packages**
+3. Edit the repo file you placed under `/etc/yum.repos.d` to correctly set up your CentOS version, by editing the baseurl field; for instance:
+
+        baseurl=https://<USERNAME>:<API_KEY>@kong.bintray.com/kong-enterprise-edition-rpm/centos/$releasever
```

Comment:
It is a bit unclear was `$releasever` is for a neophyte here... I find the CE version of this documentation much more explicit https://getkong.org/install/centos/